### PR TITLE
TF compression fix and UT

### PR DESCRIPTION
**Pull request by liuzhe-lz**
`tf.keras.Sequential` is too tricky to compress, removed from v1.8 release.

### CODE REVIEW

File: src/sdk/pynni/tests/test_compressor_tf.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+
+class CnnModel(Model):
+    def __init__(self):
+        super().__init__()
+        self.conv = Conv2D(filters=10, kernel_size=3, activation='relu')
+        self.pool = MaxPool2D(pool_size=2)
+        self.flatten = Flatten()
+        self.fc1 = Dense(units=10, activation='relu')
+        self.fc2 = Dense(units=1, activation='softmax')
```
Comment (chicm-ms) [inline src/sdk/pynni/tests/test_compressor_tf.py:66]: I suspect the output of `fc2` is always `1` since there must be a `1` if activation is` softmax`.

File: src/sdk/pynni/tests/test_compressor_tf.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+tensor1x10 = tf.constant([[1.0] * 10])
+
+pruners = {
+    'level': (lambda model: LevelPruner(model, [{'sparsity': 0.9, 'op_types': ['default']}])),
+}
+
+
+class TfCompressorTestCase(TestCase):
+
+    def test_layer_detection(self):
```
Comment (chicm-ms) [inline src/sdk/pynni/tests/test_compressor_tf.py:38]: maybe to skip the tests if tf version is 1.x ? since we have pipelines with tf 1.x installed

File: src/sdk/pynni/tests/test_compressor_tf.py (hunk 1 above)
Comment (liuzhe-lz) [reply src/sdk/pynni/tests/test_compressor_tf.py:66]: I'll fix it. But since the result of this model is never used, there should be no real difference.
The correctness of mask algorithm is checked by naive model.

Comment (chicm-ms) [reply src/sdk/pynni/tests/test_compressor_tf.py:66]: OK.

File: src/sdk/pynni/nni/compression/tensorflow/compressor.py [hunk 3]
```diff
@@ -75,6 +78,8 @@ class Compressor:
 
     def __init__(self, LayerWrapperClass, model, config_list):
         assert isinstance(model, tf.keras.Model)
+        if isinstance(model, tf.keras.Sequential):
+            raise ValueError('NNI model compression does not support `Sequential` model for now')
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/tensorflow/compressor.py:82]: why does not support `Sequential`?

**Review by chicm-ms — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

File: src/sdk/pynni/nni/compression/tensorflow/compressor.py (hunk 3 above)
Comment (liuzhe-lz) [reply src/sdk/pynni/nni/compression/tensorflow/compressor.py:82]: Because it's too hard to modify it's attributes properly. `Sequence` abuses complex properties.
